### ISSUE TITLE
Fix imports in conch for Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8507]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8507|
|Type|enhancement|
|Created|2016-06-27 06:54:49Z|
|Branch|https://github.com/twisted/twisted/tree/8507-rodrigc-conch|

There are a few places in conch where the imports need to be fixed for Python 3

<details><summary>Searchable metadata</summary>

```
trac-id__8507 8507
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8507_rodrigc_conch 8507-rodrigc-conch
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1467010489439906 1467010489439906
changetime__1467046721905390 1467046721905390
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_2"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/230

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban set owner to @rodrigc|
|-|-|

Looks good. Thank! Please see the minor comment inside the PR and then merge.

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_5"></a>|GitHub set status to `closed`|
|-|-|

In changeset 886b8b0acd964b543537c2ea23d76fa6caf174ee
```
#!CommitTicketReference repository="" revision="886b8b0acd964b543537c2ea23d76fa6caf174ee"
Merge 8507-rodrigc-conch: Fix imports in conch for Python 3

Author: rodrigc
Reviewer: adiroiban
Fixes: #8507
```
